src/Tribe/Tickets.php

```diff
@@ -1389,5 +1393,15 @@ public static function get_login_url() {
 			 */
 			return apply_filters( 'tribe_tickets_ticket_login_url', $login_url );
 		}
+
+		/**
+		 * @param $return
```

Comment:
What's the purpose of `$return` - could we add a short explanation? Not for instance quite understanding why we wouldn't do something like this at the callsite:

```
if ( $this->uncheckin( $order_id ) ) { 
    $this->maybe_update_attendees_cache(); 
}
```